.bazelci/examples_naming.yml

```diff
@@ -1,19 +1,16 @@
 common: &common
+  working_directory: ../examples/naming_package_files
   build_targets:
   - "..."
 
 tasks:
-  centos7:
+  centos:
```

Comment:
IMO, calling out the version is helpful, especially if we start testing on, e.g., multiple versions of Ubuntu.

Analogous changes are also not done in multiple places below.